content/basics/ably.textile

```diff
@@ -0,0 +1,92 @@
+* Live update services
+* Internet of Things
+* "And much more":https://ably.com/tutorials
+
+To learn more about how to use Ably, take a look at our "use cases and solutions":https://ably.com/solutions.
+
+h2(#about). API overview
+
+Ably supports both realtime and REST interfaces. The "Realtime library":/basics/use-ably#realtime-interface establishes a WebSocket connection allowing realtime messages to be efficiently sent both to and from the Ably. The "REST library":/basics/use-ably#rest-interface communicates using the HTTP protocol for all operations and is effectively stateless.
+
+There are a number of "Ably SDKs":https://ably.com/download in different programming languages to match your needs.
+
+Some of the key features are: 
+
+* "Pub/Sub with channels":#pubsub
+* "Presence and occupancy":#presence-and-occupancy
+* "Message storage and history":#message-storage-and-history
+* "Push Notifications":#push-notifications
+
+h3(#pubsub). Pub/Sub with channels
+
+Ably uses the Publish/Subscribe (Pub/Sub) architectural pattern to allow any number of publishers to publish data and any number of subscribers to subscribe to a source of data. This works with Ably's concept of channels. Channels are used to aggregate all data into a single location, allowing publishers to publish to just one location and subscribers to subscribe to just one source.
+
+Read more in our "Pub/Sub section":/channels.
+
+h3(#presence-and-occupancy). Presence and occupancy
+
+Ably’s presence feature allows clients or devices to announce their presence on a channel. Other devices or services may then subscribe to these presence events, such as entering, updating their state, or leaving the channel. You can also request a list of clients or devices that are online/offline on a channel at a particular point in time via the REST API.
+
+Occupancy provides metrics about the clients attached to a channel, including a count of the number of clients attached, the number of publishers and subscribers, the number of presence members, and the number of presence publishers and subscribers.
+
+Read more in our "Presence and occupancy section":/presence-occupancy.
+
+h3(#message-storage-and-history). Message storage and history
+
+By default messages are only stored for two minutes on Ably’s servers, these messages are retrieved automatically by the SDK in the event of a lost connection. However, if you've enabled message persistence and the disconnection is longer than two minutes you can use the History API to retrieve messages.
+
+Ably's message storage and history allows messages to be accessed after initially being sent. You can persist the last message sent using the rewind channel option or enable persistence of all messages for 24 hours for the free tier or 72 hours for paid tiers. These messages can be retrieved using the History API. You can also persist messages to external storage services such as AWS through the use of "Ably Integrations":/integrations.
```

Comment:
This is the first introduction of tiers so there needs to be a bit more information about what we are referring to. Please put, "free account tier or 72 hours for paid account tiers."